Automattic/jetpack
# Connection Register: Add current user email to connection register request

#### Changes proposed in this Pull Request:

This adds the current user's email address to send as part of the register request body. The intent here is to be able to use this information in the .com side, by adding the register to the site's audit log. We can then use this email address, along with the jetpack_version and external user ID to log all required data for ToS acceptance.

#### Jetpack product discussion


Much of this is motivated by the ToS conversation, recapped here: p9dueE-1Gc-p2, but there may be more value to storing this data later on, such as back-checking who we expect the be the "site owner", a way to communicate to someone if the connection breaks, or to follow-up with them for other reasons. 

#### Does this pull request change what data or activity we track or use?


Yes. We were not sending this email address until now, nor were we storing it. 

#### Testing instructions:






- Disconnect Jetpack
- Sandbox your connection, and point to this patch D47562-code
- Click "Set up Jetpack" - this will register the site. 
- You do not need to complete the whole user auth.
- Check the site's Report Card audit log in wordpress.com 
- Verify the email was logged accurately. 

#### Proposed changelog entry for your changes:

N/A
